### PR TITLE
Fixes #222, #229, #226

### LINKED ISSUE #222: Fire Hydrology: allow user to enter downstream distance
Allow user to input the downstream distance for the "Query by Fire Perimeters" workflow for the trace distance and use this value in the nldi_poly_query service request

### PR DESCRIPTION
**Pull request by harper-wavra**
222 - add steps in "Query by Fire Perimeters" workflow to allow users to select downstream trace distance. Add to report and print modal.
229 - Updated 'Click on the Map' text to say 'Select a delineation point on the map.'
226 - added workflow and step descriptions. You can either add text or leave blank if there is no further explanation.

### CODE REVIEW

File: src/app/content-right/report/report.component.html [hunk 1]
```diff
@@ -76,6 +76,7 @@ <h4>{{data.title}}</h4>
 						</table>
 					</div>
 					<!-- Fire Hydrology: Query by Fire Perimeters-->
+					<div *ngIf="output.key=='downstreamDist'">Downstream Trace Distance (miles): {{output.value}}</div>
```
Comment (harper-wavra) [inline src/app/content-right/report/report.component.html:79]: @Anders-Hopkins Do you know if the `downstream_dist` in the nldi_polygon_query is miles or something else? 
https://streamstats.usgs.gov/nldipolygonservices/docs#/default/query_poly_nldi_poly_query__post

File: src/app/map/map.component.spec.ts [hunk 2]
```diff
@@ -20,6 +20,7 @@ describe('MapComponent', () => {
       {
           "label": "Select a Fire Hydrology Query Method",
           "name": "selectFireHydroProcess",
+          "description":"Choose from list of fire hydrology query methods.",
```
Comment (amedenblik) [inline src/app/map/map.component.spec.ts:23]: I don't think we need a description for this step right now.

File: src/assets/workflows.json [hunk 3]
```diff
@@ -29,7 +31,7 @@
                 "options":
                 [
                     {
-                        "text":"Click on the map",
+                        "text":"Select a delineation point on the map.",
```
Comment (amedenblik) [inline src/assets/workflows.json:34]: "Select a delineation point on a blue stream line on the map." (this clarifies what a "delineation point" is and is is similar to the current StreamStats text "Use your mouse or finger to click or tap a blue stream cell on the map").

File: src/assets/workflows.json [hunk 4]
```diff
@@ -44,13 +46,14 @@
...
         "title": "Fire Hydrology",
-        "description": "Fire hydrology description here.",
+        "description": "Query fire hydrology based on delineated basin or fire perimeter.",
         "functionality": "Core",
         "icon": "far fa-fire-alt",
         "steps":
         [
             {
                 "label": "Select a Fire Hydrology Query Method",
+                "description": "Choose from list of fire hydrology query methods.",
```
Comment (amedenblik) [inline src/assets/workflows.json:56]: I don't think we need a description for this step right now.

**Review by amedenblik — CHANGES_REQUESTED** [review]

File: src/app/content-right/report/report.component.html (hunk 1 above)
Comment (Anders-Hopkins) [reply src/app/content-right/report/report.component.html:79]: @harper-wavra Distance is in kilometers: https://waterdata.usgs.gov/blog/nldi-intro/#:~:text=The%20Hydro%20Network%2DLinked%20Data%20Index%20(NLDI)%20is%20a,and%20outlets%20of%20HUC12%20watersheds.

**Review by amedenblik — APPROVED** [review]

**Pull request merged**